build/DependencyVersions.props

```diff
@@ -17,6 +17,8 @@
     <PlatformAbstractionsVersion>2.0.0-preview2-002093</PlatformAbstractionsVersion>
     <DependencyModelVersion>2.0.0-preview2-002093</DependencyModelVersion>
     <CliCommandLineParserVersion>0.1.0-alpha-142</CliCommandLineParserVersion>
+    <CliProjectJsonMigration>1.1.0-alpha-2127</CliProjectJsonMigration>

```

Comment:
Could you add more detail? I don't think there will be a diverge. After this pull request, cli-migration is the only repo publishing this package to MyGet feed. And I don't think it needs to keep up with cli repo, since it is targeted to netstandard1.5 and have fairly different concern with cli repo